### DISCUSSION TITLE
Can't connect from Windows 10

### DISCUSSION THREAD

**Discussion by DLTeamTUC** [rustdesk/rustdesk#9714] [Q&A]
Hello..

I suddenly cannot connect from my laptop with Windows 10 (I'm using Rustdesk ver. 1.3.1), the error message is "Not ready. Please check your connection.". Previously, it worked fine and I could connect to my PC with Ubuntu. On the Ubuntu PC, there is no such message.
Also, I tried pinging rs-ny.rustdesk.com and it returns "request timed out"
Could anyone please give me the solutions?

Thanks

**Comment by rustdesk** [comment]
https://www.reddit.com/r/rustdesk/comments/1cr8kfv/should_you_selfhost_a_rustdesk_server/
